remake.yml

```diff
@@ -10,15 +10,15 @@ packages:
   
 ## All R files that are used must be listed here:
 sources:
-  - scripts/loadConfig.R
+  - scripts/load_internals.R
   - scripts/load_config.R
   - scripts/lake_summary_locations.R
   - scripts/lake_summary_nldas.R
   - scripts/get_raw_nldas_nc.R
   - scripts/post_notaro_thredds.R
 
-include:
-  - remake_wqp.yml
+##include:
```

Comment:
leaving this out until we can separate these concepts and do a master `remake.yml` that will call all of the subsections